splinepy/io/npz.py

```diff
@@ -10,60 +10,108 @@
 
 import numpy as np
 
+from splinepy import io
+
 
 def load(
     fname,
 ):
     """
-    Read spline in `.npz` form.
+    Read a list of splines in `.npz` form.
 
     Parameters
     -----------
     fname: str
 
     Returns
     --------
-    dict_spline: dict
+    list_spline: list of NURBS / BSpline / Bezier / Rational Bezier
     """
     loaded = np.load(fname)
-    whatami = loaded["whatami"][0]
 
-    dict_spline = {}
-    # update weights
-    if whatami.startswith("NURBS"):
-        dict_spline.update(weights=loaded["weights"])
+    # Initialize an empty list to store the splines
+    list_of_spline_dicts = []
+
+    processed_keys = set()
```

Comment:
```suggestion
```
unused

Reply:
The processed_keys are used in
```
for key in loaded:
    if key.startswith("spline_") and key not in processed_keys:
```

Reply:
I see. Instead, can you just export number of spline and and loop range(n_splines)? This seems a bit much

Reply:
Should be fixed in [e8d3693](https://github.com/tataratat/splinepy/pull/280/commits/e8d3693c8a70f2d15d01e19017b1f7fb4f990877)

Reply:
thanks!